events/forms.py

```diff
@@ -0,0 +1,28 @@
+from django import forms
+
+
+class EventForm(forms.Form):
```

Comment:
Just to clarify, we use `forms.Form` because we want to keep events@p.o (and its Google Calendar) as primary data source for events, right?

Reply:
Exactly, we don't want to use the models as technically they are "unrelated" from the actual submission.